build/patches/Welcome-screen.patch

```diff
@@ -249,36 +214,18 @@ diff --git a/chrome/android/java/src/org/chromium/chrome/browser/firstrun/FirstR
 -                        processFreEnvironmentPreNative();
 -                    });
 -        });
-+        mIsChild = false;
-+        processFreEnvironmentPreNative();
```

Comment:
I think this is still necessary

Reply:
Yes, this took me quite a while to find for v101.